domains/python-coder.json

```
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "defPythonCoder",
+    "discord": "pyth0ncoder"
+  },
+  "records": {
+    "A": ["185.199.108.153", "185.199.109.153", "185.199.110.153", "185.199.111.153"],
+    "AAAA": ["2606:50c0:8000::153", "2606:50c0:8001::153", "2606:50c0:8002::153", "2606:50c0:8003::153"]
```

```suggestion
    "CNAME": "defpythoncoder.github.io"
```

i recommend you use this over using A/AAAA records for github pages because its neater and does the same thing